articles/server-platforms/symfony/01-login.md

```diff
@@ -11,6 +11,15 @@ This tutorial and seed project have been tested with the following:
 * Symfony 2.8
 :::
 
+<%= include('../../_includes/_package', {
```

Comment:
@mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.